components/giftbox.vue

```diff
@@ -252,6 +252,7 @@
         text-overflow: ellipsis;
         width: fit-content;
         max-width: 100%;
+        margin-bottom: 1rem;
```

Comment:
@simonpubliq I noticed your commits are a bit inconsistent in scope which makes them a bit harder to review sometimes. Make sure to read the article I sent you on commit messages, and also try to do atomic commits. This CSS change for example is totally fine on its own, but unrelated to the change in the `fetchFeatures` method. By committing changes like this separately instead, you can make it easier to revert, cherry-pick, and review each commit when needed. 😊

It should be possible to stage changes in the same file separately, so you wouldn't have to "undo" work and then "redo" it later to commit it: https://stackoverflow.com/questions/1085162/commit-only-part-of-a-file-in-git
Apps like [Fork](https://git-fork.com/) also support this and make it even easier. 
If you need help with staging only parts of your changes, definitely ping me!

Reply:
Sadly I don't have a go-to resource on atomic commits, but this article seems to cover most of it: https://curiousprogrammer.dev/blog/why-i-create-atomic-commits-in-git/